sky/clouds/aws.py

```diff
@@ -38,6 +40,22 @@ class AWS(clouds.Cloud):
     _REPR = 'AWS'
     _regions: List[clouds.Region] = []
 
+    _STATIC_CREDENTIAL_HELP_STR = (
+        'Run the following commands:'
+        '\n      $ pip install boto3'
+        '\n      $ aws configure'
+        '\n    For more info: '
+        'https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html'  # pylint: disable=line-too-long
+    )
+
+    _SSO_CREDENTIAL_HELP_STR = (
+        'Run the following commands:'
+        '\n      $ awsv2 configure sso'
+        '\n      $ awsv2 sso login --profile <profile_name>'
```

Comment:
This is tricky since skypilot will install `awscli` as the dependency. The `aws` in the path can be in `v1`, which will fail for the `configure sso` and `sso login` (only supported by `v2`).

Reply:
How about e.g.,
```
$ aws configure sso  # Must use aws v2 CLI
...
```
The reason is users may wrongly google/look for a tool called "awsv2"?

Reply:
Good point! Added the `must use aws v2 cli` to the previous line.